app/templates/playerUpgradeMode.html

```diff
@@ -0,0 +1,35 @@
+<div class="modal-content">
+  <div class="modal-header">
+    <button type="button" class="close" ng-click="cancel()" data-dismiss="modal" aria-hidden="true">&times;</button>
+    <h4 class="modal-title">Player Upgrade</h4>
+  </div>
+  <form>
+    <div class="modal-body">
+      <p>Please select upgrade mode</p>
+      <div class="radio">
+        <label>
+          <input type="radio" name="mode" ng-model="mode" value="default">
+          Upgrade V2 player to a default V7 player
+          <i class="glyphicon glyphicon-question-sign helpIcon ng-scope" tooltip="Upgrade the player to a V7 player with default plugins enabled"
+             tooltip-placement="right" tooltip-popup-delay="500" tooltip-append-to-body="false"></i>
+        </label>
+      </div>
+      <div class="radio">
+        <label>
+          <input type="radio" name="mode" ng-model="mode" value="template">
+          Upgrade V2 player to a pre-defined V7 player
+          <i class="glyphicon glyphicon-question-sign helpIcon ng-scope" tooltip="Upgrade the player to a V7 player, using a pre-configured V7 player as a template"
+             tooltip-placement="right" tooltip-popup-delay="500" tooltip-append-to-body="false"></i>
+        </label>
+      </div>
+      <div class="form-group" ng-hide="mode != 'template'">
+        <label for="playerId">Player ID</label>
+        <input type="text" class="form-control" id="playerId" ng-style="inputStyle" ng-model="playerId">
+      </div>
+    </div>
+    <div class="modal-footer">
+      <button ng-click="close(false)" class="btn btn-default">Cancel</button>
+      <button ng-click="close(true)" ng-disabled="!validate()" class="btn btn-primary">OK</button>
+    </div>
+  </form>
+</div>
```

Comment:
please add a new line at the bottom of the page

Reply:
Done.